bitcoin_client_rs/src/wallet.rs

```diff
@@ -252,6 +252,25 @@ impl core::fmt::Display for WalletPubKey {
     }
 }
 
+/// Returns true iff `descriptor_template` contains a `a:` fragment
+pub fn contains_a(descriptor_template: &str) -> bool {
```

Comment:
I think you can use `descriptor_template.contains("a:")` instead

Reply:
but multiple wrappers could be present before the `:`, so e.g. you could hav something like `ac:...`

Reply:
Ah right ! I am less qualified in miniscript, I did not know you can stack wrappers. Good to go then